EnglishFilter/sections/general_url.txt

```diff
@@ -81,6 +81,7 @@ porn*.net/bees.js|
 !+ NOT_OPTIMIZED
 /getads?
 !
+/bannerpop/uploads/*$image,script
```

Comment:
It seems modifiers are unnecessary in this case. Just `/bannerpop/uploads/*`

Reply:
https://github.com/AdguardTeam/AdguardFilters/pull/88218/commits/2023e9a687a9d5b40c3d189ee58ecddc5ff55488